convert.php

```diff
@@ -0,0 +1,72 @@
+<?php
+$languages = [
+    'Chinese' => 'zh',
```

Comment:
I don't think two letter codes are valid but I can't find what it should be either, I've asked in the translations discord

Reply:
Apparently this should be zh_TW